eZ/Publish/API/Repository/Values/Content/Content.php

```diff
@@ -28,6 +28,18 @@
     abstract public function getVersionInfo();
 
     /**
+     * Shorthand method for getVersionInfo()->getName().
+     *
+     * @see \eZ\Publish\API\Repository\Values\Content\VersionInfo::getName()
+     *
+     * @param string|null $languageCode
+     *
+     * @return string|null The name for a given language, or null if $languageCode is not set
+     *         or does not exist.
+     */
+    abstract public function getName($languageCode = null);
```

Comment:
That said, I applied the proposed change and it simplifies the patch quite a bit, including avoiding the  comment from @bdunogier  and comments from @Nattfarinn while at it.